frontend/src/metabase/visualizations/lib/settings/graph.unit.spec.js

```diff
@@ -194,3 +195,93 @@ describe("GRAPH_AXIS_SETTINGS", () => {
     );
   });
 });
+
+describe("GRAPH_DISPLAY_VALUES_SETTINGS", () => {
+  describe("graph.show_values", () => {
+    const getHidden =
+      GRAPH_DISPLAY_VALUES_SETTINGS["graph.show_values"].getHidden;
+    it("should be hidden on normalized charts without line series", () => {
+      const isHidden = getHidden(
+        [
+          { card: { display: "area" } },
+          { card: { display: "area" } },
+          { card: { display: "bar" } },
+        ],
+        {
+          series: series => ({ display: series.card.display }),
+          "stackable.stacked": "normalized",
+        },
+      );
+
+      expect(isHidden).toBe(true);
+    });
+
+    it("should be visible on normalized charts with line series", () => {
+      const isHidden = getHidden(
+        [
+          { card: { display: "area" } },
+          { card: { display: "area" } },
+          { card: { display: "line" } },
+        ],
+        {
+          series: series => ({ display: series.card.display }),
+          "stackable.stacked": "normalized",
+        },
+      );
+
+      expect(isHidden).toBe(true);
```

Comment:
```suggestion
      expect(isHidden).toBe(false);
```

Should be visible implies `isHidden = false`?